app/services/draw_starter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+#
+# Service object to handle the starting of a draw. Checks to make sure the draw
+# is in the correct status, updates the status, and sends e-mail invitations to
+# students in the draw.
+class DrawStarter
+  # Class method to permit calling :start on the class without instantiating the
+  # service object directly
+  #
+  # @attr draw [Draw] the draw in question
```

Comment:
I actually prefer this paradigm for service objects and I'm surprised that I forgot to mention it when we started implementing them. I picked it up from the Upcase forums when I was doing some of the Ruby challenges: [this]() is the thread and [here](http://blog.codeclimate.com/blog/2012/11/14/why-ruby-class-methods-resist-refactoring/)'s a great article on it. Take a look and see if it changes your mind; the reason why we wouldn't want a module is because modules don't maintain state, making refactoring and method extraction difficult, and instantiation just feels unnecessary for single-use objects like this. I really think we should stick with it moving forward and potentially even refactor our controllers and existing service objects to follow that model.